src/dataframe/dataframe.jl

```diff
@@ -1115,7 +1161,53 @@ function Base.push!(df::DataFrame, row::Union{AbstractDict, NamedTuple})
     df
 end
 
-# array and tuple like collections
+"""
+    push!(df1::DataFrame, iterable::Any)
```

Comment:
Could you merge this docstring with the one above (or rather move the one above here since this method is more general)? It would be easier to read as it would avoid repetitions. Also, `DataFrameRow` is more similar to `NamedTuple` than a random iterable.

Reply:
fixed